docs/TrafficCaptureAndReplayDesign.md

```diff
@@ -0,0 +1,271 @@
+# TrafficCapture and Replay
```

Comment:
```suggestion
# Traffic Capture and Replay Design
```
I think this document might be better classified as an overview instead of a design.

Reply:
Made the partial change - since there's a question of whether its an Overview or Design, I'll leave the filename speak for itself (the ALB splitter doc doesn't mention what kind of doc it is within the title, so I didn't bother w/ that for consistency).